get_info.php

```diff
@@ -1 +1,159 @@
-<?php?>
+<?php
+require 'db/config.php';
+require 'get_query.php';
```

Comment:
I think `query_string_generators.php` would be a better name for this file.

Reply:
Would all query generators be within this file? Would it be a good idea to split GET and POST queries into separate files? What about get_query_generators.php?